src/app/views/contact_view.js

```diff
@@ -1,6 +1,31 @@
 import Backbone from 'backbone';
+import Contact from 'app/models/contact';
+import _ from 'underscore';
+import $ from 'jquery';
 
 const ContactView = Backbone.View.extend({
+  initialize: function(options) {
+    // this.contact = options.contact;
+    this.template = options.template;
+  },
+
+  render: function() {
+    this.delegateEvents();
+    var html = this.template({name: this.model.get("name")});
```

Comment:
Error here, due to the fact that if you add a new card, the 'model' you've set for the view is a generic JavaScript object.  You'll need to make a new model when you create the view for it to work.

ex:  `newCard = new Card( { model: new Contact(contact), template: this.contactTemplate });`